files/opensesame.spec

```
@@ -126,3 +126,14 @@ description=How often to get new data (default: every 60 seconds, which is the h
 default = 6000
 type = unsigned_short
 
+[weatherstation/enable]
+description = enables/disables weatherstation
+type = boolean
+default = 0
+
+[opensensemap/box_id]
+description = Which opensensemap Box should be connected to the weather station
+
+
+[opensensemap/access_token]
```

Avoid separators other than /:

```suggestion
[weatherstation/opensensemap/token]
```